cli-utils/src/lib.rs

```diff
@@ -0,0 +1,141 @@
+use {
+    console::style, solana_clap_utils::input_validators::normalize_to_url_if_moniker,
+    solana_cli_config::Config, solana_sdk::commitment_config::CommitmentConfig, std::str::FromStr,
+};
+
+pub enum SettingType {
+    Explicit,
+    Computed,
+    SystemDefault,
+}
+
+pub fn println_name_value_or(name: &str, value: &str, setting_type: SettingType) {
+    let description = match setting_type {
+        SettingType::Explicit => "",
+        SettingType::Computed => "(computed)",
+        SettingType::SystemDefault => "(default)",
+    };
+
+    println!(
+        "{} {} {}",
+        style(name).bold(),
+        style(value),
+        style(description).italic(),
+    );
+}
+
+pub struct ConfigInput {
```

Comment:
Move this to `cli-config/`?

Reply:
I did start down that path, but then it pulls in a dependency on solana_sdk. The `solana-cli-config` is pretty simple and straightforward right now; only does this:
https://github.com/solana-labs/solana/blob/e14933c54db16812803ec51c96c1a31ff0a6bef5/cli-config/src/lib.rs#L1

Thoughts on the extra bulk in `solana-cli-config`?

Reply:
Seems ok to me, solana-sdk is almost certainly already in the dependency graph anyway.  `clap-utils/`, another alternative I thought of, doesn't feel quite right.